extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -166,3 +166,61 @@
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+/** Inline attachment specifications */
+.i-amphtml-story-inline-page-attachment-chip {
+  display: flex !important;
+  position: relative !important;
```

Comment:
Might not need this.

Reply:
position isn't needed, display is (or else the images display in a single column instead of a row)